mep/people/templates/people/member_card_detail.html

```diff
@@ -19,26 +19,52 @@
 
 <div class="card-image-viewer">
     <div class="viewer">
-        {% if prev_card %}
+        {% if page_obj.has_previous %}
         <a class="prev" rel="prev" aria-label="previous card"
-           href="{% url 'people:member-card-detail' member.slug prev_card %}"></a>
+           href="{% url 'people:member-card-detail' member.slug prev_card_id %}"></a>
         {% else %}
         <span class="prev"></span>
         {% endif %}
         <picture>
         {% with card.image|iiif_image:"size:width=430" as 1xthumbnail %}
             <source srcset="{{ 1xthumbnail }}, {{ card.image|iiif_image:"size:width=860" }} 2x">
-            <img src="{{ 1xthumbnail }}" alt="">
+            <img src="{{ 1xthumbnail }}" alt="{{ member.firstname_last }} {{ label }} card" aria-describedby="card-counter">
+            <span id="card-counter" aria-hidden="true">
+                {{ page_obj.number }}
+                <span aria-label="of">/</span>
+                {{ page_obj.paginator.num_pages }}
+            </span>
         {% endwith %}
         </picture>
-        {% if next_card %}
+        {% if page_obj.has_next %}
         <a class="next" rel="next" aria-label="next card"
-           href="{% url 'people:member-card-detail' member.slug next_card %}"></a>
+           href="{% url 'people:member-card-detail' member.slug next_card_id %}"></a>
         {% else %}
         <span class="next"/></span>
         {% endif %}
     </div>
 
+    <div class="card nav container">
+        <div class="wrapper"> {% spaceless %}
+            <nav class="cards" aria-label="cards">
+                <ol>
+                    {% for card_ in cards.all %}
```

Comment:
Can we find a better variable name than `card_` here? Kind of hard to distinguish `card_` and `card`.  Thumbnail? Canvas?

Reply:
I struggled with this, but I like Canvas!